addons/website_slides_survey/static/src/scss/website_slides_survey_result.scss

```diff
@@ -0,0 +1,5 @@
+// Certification Share modal on survey result page
+
+#wrapwrap .modal {
```

Comment:
Even if it is used only in survey_assets, people finding this one may miss an heartbeat ...

Could be great to
1/ at least add a comment
2/ see if namespacing is possible